extensions/2.0/Khronos/KHR_texture_procedurals/README.md

```diff
@@ -0,0 +1,1351 @@
+# KHR\_texture\_procedurals
+
+## Contributors
+
+- Ashwin Bhat, Autodesk, ashwin.bhat@autodesk.com
+- Alexey Knyazev, Individual Contributor, [@lexaknyazev](https://github.com/lexaknyazev)
+- Bernard Kwok, Individual Contributor, [@kwokcb](https://github.com/kwokcb)  
+- Ed Mackey, AGI, [@emackey](https://twitter.com/emackey)
+- Lutz Kettner, NVIDIA, lkettner@nvidia.com
+- Richard Sahlin, IKEA [@rsahlin](https://github.com/rsahlin)
+- Timothy Blut, NVIDIA, tblut@nvidia.com
+
+Copyright 2024 The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension provides a standardized way to represent procedural graphs which can be mapped to material channels in glTF.
+
+### Table of Contents
+
+- [Motivation](#motivation)
+- [Definitions](#definitions)
+- [Extension Declaration](#extension-declaration)
+- [Representation](#representation)
+  - [Data Types](#data-types)
+  - [Procedural Graphs](#procedural-graphs)
+  - [Procedural Nodes](#procedural-nodes)
+  - [Node Graph Connections](#node-graph-connections)
+  - [Material Binding](#material-binding)
+  - [Resource Binding](#resource-binding)
+    - [Uniform Binding](#uniform-binding)
+    - [Texture Binding](#texture-binding)
+    - [Input Stream Binding](#input-stream-binding)
+  - [Procedural Definitions](#procedural-definitions)
+    - [Structure](#structure)
+- [Structure Review](#structure-review)
+- [JSON Schema](#json-schema)
+- [Checkerboard Example](#checkerboard-example)
+- [Resources](#resources)
+
+### Motivation
+
+Textures represented as procedural graphs provides a way to extend the capabilities of glTF materials beyond what is possible with traditional texture maps. Key objectives includes:
```

Comment:
Please reword to:
`Images represented as procedural operators or functions provides a way to extend the capabilities of glTF textures beyond what is possible with bitmap based images.`